x-pack/legacy/plugins/lens/public/drag_drop/drag_drop.tsx

```diff
@@ -86,7 +108,9 @@ export function DragDrop(props: Props) {
       return;
     }
 
-    e.dataTransfer.setData('text', 'dragging');
+    // We only can reach the dragStart method if the element is draggable,
+    // so we know we have DraggableProps if we reach this code.
+    e.dataTransfer.setData('text', (props as DraggableProps).label);
```

Comment:
Is it actually required to provide draggable text? It looks like we are allowed to set custom data without a fallback to text, which makes sense because the text is not used for any of our purposes: https://developer.mozilla.org/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Drag_operations#dragdata

Reply:
Hey @wylieconlon sorry I am not following this one - what do you mean 'allowed to set custom data without a fallback to text'? Could you write an explicit example here?